<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>410</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>516</width>
    <height>410</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>516</width>
    <height>410</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Inter</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Sistema Clinica</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(220, 255, 243);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>361</width>
      <height>221</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../Downloads/hospital-clinic-building-3d-icon-illustration-png.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 48pt &quot;Niagara Solid&quot;;</string>
    </property>
    <property name="text">
     <string>CENTRO MEDICO ZEGEL</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>516</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRegistro">
    <property name="font">
     <font>
      <family>Inter</family>
     </font>
    </property>
    <property name="title">
     <string>Registro</string>
    </property>
    <addaction name="actionMedicos"/>
    <addaction name="actionEspecialidades"/>
    <addaction name="actionAreas"/>
   </widget>
   <widget class="QMenu" name="menuAcerca_de">
    <property name="font">
     <font>
      <family>Inter</family>
     </font>
    </property>
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAcerca_de"/>
   </widget>
   <addaction name="menuRegistro"/>
   <addaction name="menuAcerca_de"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionComprobantes">
   <property name="text">
    <string>Comprobantes</string>
   </property>
  </action>
  <action name="actionMedicos">
   <property name="icon">
    <iconset>
     <normaloff>../View/icon/doctor.png</normaloff>../View/icon/doctor.png</iconset>
   </property>
   <property name="text">
    <string>Médicos</string>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
    </font>
   </property>
  </action>
  <action name="actionEspecialidades">
   <property name="icon">
    <iconset>
     <normaloff>../View/icon/especialidades.png</normaloff>../View/icon/especialidades.png</iconset>
   </property>
   <property name="text">
    <string>Especialidades</string>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
    </font>
   </property>
  </action>
  <action name="actionAreas">
   <property name="icon">
    <iconset>
     <normaloff>../View/icon/area.png</normaloff>../View/icon/area.png</iconset>
   </property>
   <property name="text">
    <string>Áreas</string>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
    </font>
   </property>
  </action>
  <action name="actionProveedor">
   <property name="icon">
    <iconset>
     <normaloff>../View/images/proveedor.png</normaloff>../View/images/proveedor.png</iconset>
   </property>
   <property name="text">
    <string>Proveedor</string>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
    </font>
   </property>
  </action>
  <action name="actionVista_previa">
   <property name="text">
    <string>Vista previa</string>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
    </font>
   </property>
  </action>
  <action name="actionAcerca_de">
   <property name="text">
    <string>Acerca de</string>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
    </font>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
